<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="phonkText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>The Most Popular</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="hensonnButton">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>80</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/Hensonn.jpg</normaloff>Images/Hensonn.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="imagineButton">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>344</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ImagineDragons.jpg</normaloff>Images/ImagineDragons.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="arturButton">
    <property name="geometry">
     <rect>
      <x>516</x>
      <y>344</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/АртурПирожков.jpg</normaloff>Images/АртурПирожков.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="valeraButton">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>344</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ВалерийМиладзе.jpg</normaloff>Images/ВалерийМиладзе.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="weekendButton">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>344</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/TheWeekend.jpg</normaloff>Images/TheWeekend.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="interworldButton">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>80</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/InterWorld.jpg</normaloff>Images/InterWorld.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="acrossButton">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>80</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/AcrossTheSpiderVerse.jpg</normaloff>Images/AcrossTheSpiderVerse.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="intoButton">
    <property name="geometry">
     <rect>
      <x>516</x>
      <y>80</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/IntoTheSpiderVerse.jpg</normaloff>Images/IntoTheSpiderVerse.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>177</width>
      <height>177</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="phonkText_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Phonk Artists</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cartoonText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>5</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>The Most Popular</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cartoonText_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Cartoon Albums</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="engSingerText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>264</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>The Most Popular</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="engSingerText_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>English Singers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rusSingerText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>264</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>The Most Popular</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rusSingerText_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Russian Singers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/stars.jpg</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>phonkText</zorder>
   <zorder>hensonnButton</zorder>
   <zorder>imagineButton</zorder>
   <zorder>arturButton</zorder>
   <zorder>valeraButton</zorder>
   <zorder>weekendButton</zorder>
   <zorder>interworldButton</zorder>
   <zorder>acrossButton</zorder>
   <zorder>intoButton</zorder>
   <zorder>phonkText_2</zorder>
   <zorder>cartoonText</zorder>
   <zorder>cartoonText_2</zorder>
   <zorder>engSingerText</zorder>
   <zorder>engSingerText_2</zorder>
   <zorder>rusSingerText</zorder>
   <zorder>rusSingerText_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Images/background.qrc"/>
 </resources>
 <connections/>
</ui>
